extensions/browser/brave_extension_provider.cc

```diff
@@ -123,7 +123,12 @@ bool BraveExtensionProvider::MustRemainDisabled(
     const Extension* extension,
     disable_reason::DisableReason* reason,
     base::string16* error) const {
-  return extension->id() == brave_extension_id;
```

Comment:
Is this right condition that returns true for |brave_extension_id|?

Reply:
Right, seems like maybe this was meant to go in `MustRemainEnabled`? I think @bbondy would know for sure.

Reply:
that's fixed in a branch I'm doing now for pdfjs, I don't think it causes any problems though so I think there's no urgency to do it before this branch.